### PR TITLE
Attachment on contact us

### LINKED ISSUE #913: Terra Contact button - Attach screenshot ability
Users using the in-app contact button would like a way to attach screenshots to their question/bug/feature request to make it easier to explain to us what they want.

### PR DESCRIPTION
**Pull request by katevoss**
Fixes #913 

A user can:
- Upload one attachment, max file size is 20MB
- Click or drag to upload an attachment
- Click or drag to upload a different attachment
- Remove an attachment

### CODE REVIEW

**Review by elizzhao — APPROVED** [review]
works fine on my system!

File: src/components/SupportRequestModal.js [hunk 1]
```diff
@@ -11,6 +12,7 @@ import * as Forms from 'src/libs/forms'
 import * as Utils from 'src/libs/utils'
 import { Component } from 'src/libs/wrapped-components'
 import validate from 'validate.js'
+import Dropzone from 'react-dropzone'
```
Comment (MattPutnam) [inline src/components/SupportRequestModal.js]: nit: alphabetize

File: src/components/SupportRequestModal.js [hunk 2]
```diff
@@ -36,81 +38,146 @@ const SupportRequestModal = Utils.connectAtom(authStore, 'authState')(class Supp
...
     return !(firstName === 'N/A' || firstName === undefined)
   }
 
+  async uploadFile(files) {
+    try {
+      this.setState({ uploadingFile: true })
+      const attachmentRes = await Ajax().User.uploadAttachment(files[0])
+      const attachmentToken = attachmentRes.token
+      const attachmentName = attachmentRes.attachment.file_name
+      this.setState({ attachmentToken: attachmentToken, uploadingFile: false, attachmentName: attachmentName })
```
Comment (MattPutnam) [inline src/components/SupportRequestModal.js]: Use abbreviated object syntax: `{ attachmentToken, attachmentName, uploadingFile: false }`

**Review by MattPutnam — APPROVED** [review]
👍 Looks good, didn't test.

**Pull request merged**